frontend/styles/Home.module.css

```diff
@@ -58,11 +58,15 @@
     flex-direction: column;
     padding: 7rem 2rem 2rem 2rem;
   }
+  @media screen and (min-width: 1700px) {
+    padding: 0rem 0rem 0rem 10rem;
+    gap: 10em;
```

Comment:
this can be changed to something more responsive (if you set width to like 2000/3000px++ you can still see the space).